pairing/bls12381/gnark/gt.go

```diff
@@ -0,0 +1,105 @@
+package gnark
+
+import (
+	"crypto/cipher"
+	"io"
+	"math/big"
+
+	bls12381 "github.com/consensys/gnark-crypto/ecc/bls12-381"
+	"go.dedis.ch/kyber/v4"
+)
+
+var gtBase *bls12381.GT
+
+func init() {
+	_, _, g1, g2 := bls12381.Generators()
+	gt, err := bls12381.Pair([]bls12381.G1Affine{g1}, []bls12381.G2Affine{g2})
```

Comment:
Hm. Not a fan of doing this on start, but I guess circl does it as well. Ideally we'd pre-compute this and write a test.